cg/cli/get.py

```diff
@@ -18,18 +19,18 @@
 @click.pass_context
 def get(context: click.Context, identifier: str):
     """Get information about records in the database."""
-    context.obj["status_db"] = Store(context.obj["database"])
-    if identifier and re.match(r"^[A-Z]{3}[0-9]{4,5}[A-Z]{1}[1-9]{1,3}$", identifier):
-        context.invoke(sample, sample_ids=[identifier])
-    elif identifier and re.match(r"^[a-z]*$", identifier):
-        # try family information
-        context.invoke(family, family_ids=[identifier])
-    elif identifier and re.match(r"^[HC][A-Z0-9]{8}$", identifier):
-        # try flowcell information
-        context.invoke(flowcell, flowcell_id=identifier)
-    elif identifier:
-        LOG.error(f"{identifier}: can't predict identifier")
-        context.abort()
+    if identifier:
```

Comment:
can get rid of identifier by making it mandatory arg

Reply:
Since this is a `group`-command I guess it should be possible to run the subcommands without the `-I` flag